configure.ac

```diff
@@ -1315,21 +1330,8 @@ if test "$enable_fuzz" = "yes"; then
      ]],[[
       */ int not_main() {
      ]])])
-else
-  BITCOIN_QT_INIT
 
-  dnl sets $bitcoin_enable_qt, $bitcoin_enable_qt_test, $bitcoin_enable_qt_dbus
```

Comment:
moving this seems unrelated?

Reply:
It looks like a way the diff has been rendered. The actual diff is moving `AX_CHECK_LINK_FLAG` stuff.